R/tm_g_patient_profile.R

```diff
@@ -62,67 +62,30 @@
 #' @export
 #'
 #' @examples
-#' library(nestcolor)
+#' data <- teal.data::cdisc_data() |>
+#'   within(library(nestcolor)) |>
+#'   within(ADSL <- osprey::rADSL) |>
+#'   within(ADAE <- osprey::rADAE %>% mutate(ASTDT = as.Date(ASTDTM), AENDT = as.Date(AENDTM))) |>
+#'   within(ADCM <- osprey::rADCM %>% mutate(ASTDT = as.Date(ASTDTM), AENDT = as.Date(AENDTM))) |>
+#'   # The step below is to pre-process ADCM to legacy standard
+#'   within(ADCM <- ADCM %>% select(-starts_with("ATC")) %>% unique()) |>
+#'   within(ADRS <- osprey::rADRS %>% mutate(ADT = as.Date(ADTM))) |>
+#'   within(ADEX <- osprey::rADEX %>% mutate(ASTDT = as.Date(ASTDTM), AENDT = as.Date(AENDTM))) |>
+#'   within(ADLB <- osprey::rADLB %>% mutate(ADT = as.Date(ADTM), LBSTRESN = as.numeric(LBSTRESC)))
 #'
-#' ADSL <- osprey::rADSL
-#' ADAE <- osprey::rADAE %>%
-#'   mutate(
-#'     ASTDT = as.Date(ASTDTM),
-#'     AENDT = as.Date(AENDTM)
-#'   )
-#' ADCM <- osprey::rADCM %>%
-#'   mutate(
-#'     ASTDT = as.Date(ASTDTM),
-#'     AENDT = as.Date(AENDTM)
-#'   )
-#'
-#' # The step below is to pre-process ADCM to legacy standard
-#' ADCM <- ADCM %>%
-#'   select(-starts_with("ATC")) %>%
-#'   unique()
+#' teal.data::datanames(data) <- c("ADSL", "ADAE", "ADCM", "ADRS", "ADEX", "ADLB")
+#' teal.data::join_keys(data) <- teal.data::default_cdisc_join_keys[teal.data::datanames(data)]
 #'
-#' ADRS <- osprey::rADRS %>%
-#'   mutate(ADT = as.Date(ADTM))
-#' ADEX <- osprey::rADEX %>%
-#'   mutate(
-#'     ASTDT = as.Date(ASTDTM),
-#'     AENDT = as.Date(AENDTM)
-#'   )
-#' ADLB <- osprey::rADLB %>%
-#'   mutate(
-#'     ADT = as.Date(ADTM),
-#'     LBSTRESN = as.numeric(LBSTRESC)
-#'   )
+#' ADSL <- data[["ADSL"]]
+#' ADAE <- data[["ADAE"]]
+#' ADCM <- data[["ADCM"]]
+#' ADRS <- data[["ADRS"]]
+#' ADEX <- data[["ADEX"]]
+#' ADLB <- data[["ADLB"]]
```

Comment:
`ADSL` is the only binding used in code below
```suggestion
#' ADSL <- data[["ADSL"]]
```